x-pack/plugins/cases/public/components/case_view/components/case_view_activity.test.tsx

```diff
@@ -162,19 +168,21 @@ describe('Case View Page activity tab', () => {
   beforeEach(() => {
     jest.clearAllMocks();
     appMockRender = createAppMockRenderer();
+
     useGetCaseUsersMock.mockReturnValue({ isLoading: false, data: caseUsers });
+    useCasesFeaturesMock.mockReturnValue(useGetCasesFeaturesRes);
   });
 
   it('should render the activity content and main components', async () => {
     appMockRender = createAppMockRenderer({ license: platinumLicense });
     appMockRender.render(<CaseViewActivity {...caseProps} />);
 
-    expect(screen.getByTestId('case-view-activity')).toBeInTheDocument();
-    expect(screen.getAllByTestId('user-actions-list')).toHaveLength(2);
-    expect(screen.getByTestId('case-tags')).toBeInTheDocument();
-    expect(screen.getByTestId('cases-categories')).toBeInTheDocument();
-    expect(screen.getByTestId('connector-edit-header')).toBeInTheDocument();
-    expect(screen.getByTestId('case-view-status-action-button')).toBeInTheDocument();
+    expect(await screen.findByTestId('case-view-activity')).toBeInTheDocument();
+    expect(await screen.findAllByTestId('user-actions-list')).toHaveLength(2);
+    expect(await screen.findByTestId('case-tags')).toBeInTheDocument();
+    expect(await screen.findByTestId('cases-categories')).toBeInTheDocument();
+    expect(await screen.findByTestId('connector-edit-header')).toBeInTheDocument();
+    expect(await screen.findByTestId('case-view-status-action-button')).toBeInTheDocument();
 
     await waitForComponentToUpdate();
```

Comment:
Here and in many others we have this at the end of the test, why?

Reply:
If I remember correctly we had it to add to remove some warnings regarding the state getting updated after the end of the test. You can read more about it here https://kentcdodds.com/blog/fix-the-not-wrapped-in-act-warning